Guideline-IT-Security_EN.md

```diff
@@ -6,15 +6,15 @@
 
 The objective of this guideline is to provide medical device manufacturers and notified bodies with instructions and a specific checklist in order to:
 
-- Explain what notified bodies&#39; expectations are
```

Comment:
I think the `'&#39;'` (aka apostrophe) was set intentionally, as `bodies` (=plural) needs also the genitive here - as we talk of the expectations of the notified bodies. Having a word ending with an `s` the apostrophe seems right to me. I would vote to keep the original `bodies'`.

Reply:
Indeed